docs/docs/contribute/code/getting-started.md

```diff
@@ -6,21 +6,60 @@ license: CC-BY-SA-4.0
 # Getting started
 
 ## Stack components
-The Fragcolor ecosystem has the primary components (each in a different state of development right now):
+The Fragcolor ecosystem has a few primary components (each in a different state of development right now):
 
-- Chainblocks open-scripting programming language
-- Hasten creativity engine (to build apps, games and open worlds)
-- Clamor protocol and networking stack
+- [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) open-scripting programming language
+- [Hasten](https://github.com/fragcolor-xyz/hasten) creativity engine (to build apps, games and open worlds)
+- [Clamor](https://github.com/fragcolor-xyz/clamor) protocol and networking stack
 
-## How to start
+To contribute code to any of the above projects you'll need to setup your development environment to work with Git, Github, a code editor, Rust, & C++.
 
-You need to have Git, a GitHub account, and a code editor (we recommend VS Code).
+## Development environment
 
-*If you're not familiar with Git and the GitHub workflow please read the [Installations](https://docs.fragcolor.xyz/docs/01.installations/) and [Making changes](https://docs.fragcolor.xyz/docs/02.making-changes/) sections in the Docs Contribution guide first.*
+### Setup Git & Github
+To install Git on your machine (for Windows, Linux, & Mac) refer to the [Git installation](https://docs.fragcolor.xyz/contribute/docs/installation/#git) section.
 
-To contribute code to any of these programs you need to build them from sources on your local machine.
+You'll also need an active [Github](https://github.com/) account.
```

Comment:
Cloning using the `https` protocol for git doesn't require to have any account. They won't be able to fork and/or contribute code without it though.

Reply:
Rewrote this sentence to clarify this.